app/assets/javascripts/analytics/scroll-tracker.js

```diff
@@ -35,6 +35,17 @@
       ['Heading', '11. Your charity’s legal structure and what it means'],
       ['Heading', '12. Charity officers - the chair and treasurer'],
       ['Heading', '13. Technical terms used in this guidance']
+    ],
+    '/guidance/universal-credit-how-it-helps-you-into-work': [
+      ['Heading', 'Support from your work coach'],
+      ['Heading', 'Help available for parents'],
+      ['Heading', 'When you can claim Universal credit'] 
```

Comment:
A comma is missing at the end and the heading is slightly wrong as "Universal Credit" has a **capital C** on that page.
(I will fix this and push in a bit, just wanted to leave a comment so you are aware.)